meson_options.txt

```diff
@@ -21,6 +21,8 @@ option('directwrite', type: 'feature', value: 'disabled',
   description: 'Enable DirectWrite shaper backend on Windows (experimental)')
 option('coretext', type: 'feature', value: 'disabled',
   description: 'Enable CoreText shaper backend on macOS')
+option('wasm', type: 'feature', value: 'disabled',
+  description: 'Enable WebAssembly shaper backend')
```

Comment:
Awesome work. Since this apparently will use a generic 'Wasm' table tag from a font maybe it can be named a 'font backend' in hope someday it gains outline drawing / painting capabilities.